keyboards/boardwalk/config.h

```diff
@@ -21,8 +21,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define VENDOR_ID       0xCDCD
 #define PRODUCT_ID      0x5337
 #define DEVICE_VER      0x0001
-#define MANUFACTURER    shens
-#define PRODUCT         Boardwalk
+#define MANUFACTURER    shensmobile
+#define PRODUCT         shensmobile Boardwalk
```

Comment:
```suggestion
#define PRODUCT         Boardwalk
```

Reply:
I changed this based on a suggestion from the VIA docs (linked in the commit body), but I can revert this if you insist.

Also did you want the whole changeset reverted, or just the Product line?

Reply:
Just `PRODUCT`. IMO it doesn't make sense to combine them as that's exactly what `MANUFACTURER` is for.

Reply:
But this is not what Windows shows in the devices, it only shows `PRODUCT` so I suggest making `PRODUCT` the full name you want shown, which may be the make and model.

Reply:
That's just Windows though - the QMK Toolbox will display "shensmobile shensmobile Boardwalk" 👎

Reply:
I'm feeling like this is a problem better solved in QMK Toolbox / VIA Configurator. Windows is gonna Windows.